ApollosProject/apollos-prototype
# Adds clarity around current position in devotional list

## DESCRIPTION
![2019-08-08 13 32 06](https://user-images.githubusercontent.com/1637694/62725020-48677d00-b9e2-11e9-9c27-f8de18beda6b.gif)

### What does this PR do, or why is it needed?

This PR is a 80/20 approach to bring clarity to a user's position in a list of devotional items. 

I've done two things to accomplish this: 

1. I've made sure that the current devotional you are on displays in the list of devotionals. That way the number ordering should always make sense. 

2. Added support for scrolling to the current item in the horizontal list, if the current item is found in the horizontal list. 

### What design trade-offs/decisions were made?

Did a really ugly hack, copied over the width of the `TileCard` into the `HorizontalTileFeed` as a constant. @burkeshartsis or @conrad-vanl let me know if you know of a better solution. 

### How do I test this PR?

1. Open the app.
2. Visit a devotional with multiple sbiliing content items. 
3. Click to the next content item at the bottom. The ordering should make sense and the new item should load with the scroll moved to the current item.

### What automated tests did you write?

## TODO:

- [x] Closes #958 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
